pre_commit_config_updater.py

```diff
@@ -1,6 +1,22 @@
 import argparse
+import shutil
 import urllib.request
 
+import toml
+
+
+def init_toml(toml_file: str):
```

Comment:
can there be a case that we use this pre-commit config updater where this file doesn't exist?

Reply:
Nope

Reply:
I think it is a pep standard now but are we sure it always exists.

Reply:
yes it's standard now and even if it's not there it will be auto created for `issort` and `black` config.